readthedocs/projects/models.py

```diff
@@ -238,8 +238,8 @@ class AddonsConfig(TimeStampedModel):
 
     # Notifications
     notifications_enabled = models.BooleanField(default=True)
-    notifications_show_on_latest = models.BooleanField(default=True)
-    notifications_show_on_non_stable = models.BooleanField(default=True)
+    notifications_show_on_latest = models.BooleanField(default=False)
+    notifications_show_on_non_stable = models.BooleanField(default=False)
```

Comment:
This is a place where I wish we had some kind of `null` default, because there's no way to write a migration that migrates this for folks that wanted to specify `on` in some fashion, but I think that's OK?

Reply:
Last time I wrote a queryset where I assumed if the modified and created dates are the same, then it wasn't changed by the user, but here I guess it may be surprising for some people that actually want this on, but maybe they just didn't know they could turn it off.

Reply:
I don't think we have modified data on the booleans though, only the overall model which has a bunch of unrelated stuff on it :/